package.json

```diff
@@ -6,14 +6,17 @@
   "dependencies": {
     "clean-css": "^4.2.1",
     "eleventy-plugin-nesting-toc": "^1.1.0",
-    "markdown-it-anchor": "^5.2.4"
+    "markdown-it-anchor": "^5.2.4",
+    "node-fetch": "^2.6.0"
   },
   "devDependencies": {
     "@11ty/eleventy": "^0.8.2"
   },
   "scripts": {
     "build": "npx eleventy",
-    "serve": "npx eleventy --serve"
+    "serve": "npx eleventy --serve",
+    "update-data": "node fetchData.js",
+    "full-build": "npm run update-data && npm run build"
```

Comment:
`build` vs `full-build` is a strange distinction to me.  Why would I ever *not* want to do a full build?  Does `build` only build half the site? (rhetorical questions)  I think `update-build`, or `update-and-build` makes more sense because it describes exactly what it is doing.  Definitely not a big deal either way, just my initial reaction.  If `full-build` makes more sense to you, I'm sure I'll get used to it.

Reply:
yeah, agreed.  Anything adequately descriptive seemed to garble on too long.  `Update-Build`  still sounds like you're just updating the build and I wouldn't guess what it did any better.  I do like `update-and-build` to suggest you're doing two things sequentially.

Reply:
Oh good point.  `update-build` is just as open to misinterpretation.  Good names are so hard!